### PR TITLE
Add Chinese Simplified locale file

### PR DESCRIPTION
**Pull request by FutabaWasuke**

### CODE REVIEW

File: public/locales/zh/language.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "Language": "Language",
+    "zh": "Chinese",
```
Comment (SetZero) [inline public/locales/zh/language.json:3]: For this file I have the format:
- `lang` (e.g. `zh`): The language name in the language (`zh` -> `中文`, `en` -> `English`, `de` -> `Deutsch`)
- `lang native` (e.g. `zh native`): The language in the current selected language (`zh` ->  `中文`, `en` -> `英文`)

`dev` is the "language" for development, which is mostly used to find untranslated text. This replaces all strings with "---"
`Language` is used as the Header for the language selection

File: public/locales/zh/common.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "Open In Browser": "在浏览器打开",
+    "Muted": "已静音",
+    "Deafened": "已关闭声音",
+    "Joined": "已加入",
+    "User ID": "用户 ID",
+    "Settings": "设置",
+    "None": "无",
+    "Image too large": "[[ 图片太大了捏 ( {{size}} 最大体积是 {{maximum}}) ]]",
+    "write something": "发点什么吧 :D",
+    "Feature Not Implemented": "{{feature}} (Not Implemented)",
```
Comment (SetZero) [inline public/locales/zh/common.json:19]: I use this string to mark that something is not implemented, e.g. "Autopilot (Not Implemented)"

**Review by SetZero — COMMENTED** [review]
Hi! Thanks for your PR. I just added some minor comments to some untranslated text. If you want, you can fix them.

**Comment by SetZero** [comment]
Pipeline is failing, because I forgot to remove a step from the pipeline that is signing the binary. 
This can be ignored.

**Comment by FutabaWasuke** [comment]
Good luck in the year of the dragon, I apologize for the delay due to the holiday. 
I have updated language.json, please take a look.

**Review by SetZero — APPROVED** [review]
Thank you for your work. Now it looks good to me!

**Comment by SetZero** [comment]
> Good luck in the year of the dragon, I apologize for the delay due to the holiday. I have updated language.json, please take a look.

Thank you and good luck to you, too. No worries about the delay, we don't have any time pressure here :D

**Pull request merged**